### PR TITLE
Update from Specification Editors

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**
To be able to send the August newsletter I either need an update from the specification editors or permission from the Editors to remove this section from the newsletter this month.

### CODE REVIEW

**Comment by RubenVerborgh** [comment]
This should be an issue, not a PR, and should be assigned to editors as assignee, not reviewer (nothing to review).

File: _posts/newsletter/2020-08-06-this-month-in-solid.md [hunk 1]
```diff
@@ -38,3 +38,5 @@ This month, there's been a website refresh, [check out the latest version of the
 * [Contact Inrupt careers](https://inrupt.com/careers) 
 
 ### Update from the [Specification Editors](https://github.com/solid/process/blob/master/editors.md)
+
+(insert update)
```
Comment (csarven) [inline _posts/newsletter/2020-08-06-this-month-in-solid.md:42]:
```suggestion

* Key requirements pertaining to read and write operations are included in the Editors Draft: https://solid.github.io/specification/
* Updates to Use Cases and Requirements for Authorization: https://github.com/solid/authorization-panel/blob/master/proposals/wac-ucr/index.bs and Data Interoperability: https://github.com/solid/data-interoperability-panel/blob/master/proposals/use-cases-and-requirements/index.bs
* Progress on Proposals to Authentication: https://github.com/solid/authentication-panel/blob/master/oidc-authentication.md , Data Interop: https://github.com/solid/data-interoperability-panel/tree/master/proposals/ecosystem
* General exchange concerning existing Authorization issues as well as considerations for new functionality: https://github.com/solid/authorization-panel/issues

Join us in the W3C Solid Community Group. Links to activities are available at https://www.w3.org/community/solid/wiki/Main_Page
```

**Comment by Mitzi-Laszlo** [comment]
Thank you @csarven :)

**Comment by csarven** [comment]
This is a recurring update that we, the Editors, need to provide. At the very least indicate that we don't have an update for the month in a timely manner. We can have a fallback, in case no update text is provided. I suggest to either exclude the section or have a default text along the lines of "Join us.." (eg. https://github.com/solid/solidproject.org/pull/302/files#diff-8a45f8b2afcce5c0c42bcd8b3164bfebR47 ). I think the latter is useful for anyone seeing the newsletter for the first time and may jump in.

**Pull request merged**